conbench/entities/run.py

```diff
@@ -698,18 +721,7 @@ class _RunFacadeSchemaCreate(marshmallow.Schema):
     github = marshmallow.fields.Nested(
         SchemaGitHubCreate(),
         required=False,
-        metadata={
-            "description": conbench.util.dedent_rejoin(
-                """
-                Use this object to tell Conbench with which commit the
-                Run/BenchmarkResult is associated.
-
-                This field can be left out for testing purposes. Note however
-                that commit information is required for powering valuable
-                Conbench capabilities.
```

Comment:
So far we had this (insufficient) spec for `run.github`. Updated spec for both `run.github` and `result.github`. This is of course again a temporary state, but one that is better than before.